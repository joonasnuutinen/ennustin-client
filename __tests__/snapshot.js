import React from "react";
import renderer from "react-test-renderer";
import Index from "../pages/index";
import Profile from "../pages/profiili";

it("renders homepage unchanged", () => {
  const tree = renderer.create(<Index />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders profile page unchanged", () => {
  const profile = renderer.create(<Profile />).toJSON();
  expect(profile).toMatchSnapshot();
});
