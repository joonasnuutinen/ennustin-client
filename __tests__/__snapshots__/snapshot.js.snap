// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
Array [
  <header
    className="flex justify-center items-center w-full h-12 px-2 text-white z-10"
  />,
  <div
    className="min-h-screen"
  >
    <main
      className="jsx-2035999944 flex flex-col flex-1 justify-center items-center w-full pb-20 px-2 font-light"
    >
      <img
        alt="Ennustin"
        className="jsx-2035999944 w-full max-w-sm"
        src="/ennustin.svg"
      />
      <p
        className="jsx-2035999944 py-8 text-center text-2xl"
      >
        Pilvipohjainen työkalu asioiden ennustamiseen
      </p>
      <p
        className="jsx-2035999944"
      >
        Lisätietoja:
         
        <a
          className="jsx-2035999944 font-normal"
          href="mailto:info@autua.fi"
        >
          info@autua.fi
        </a>
      </p>
    </main>
    <footer
      className="jsx-2035999944 flex justify-center items-center h-16 w-full px-2 bg-white"
    >
      Palvelun tarjoaa
       
      <a
        className="jsx-2035999944"
        href="https://autua.fi"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Autua"
          className="jsx-2035999944 ml-2 h-4"
          src="/autua.svg"
        />
      </a>
    </footer>
  </div>,
]
`;

exports[`renders profile page unchanged 1`] = `
Array [
  <header
    className="flex justify-center items-center w-full h-12 px-2 text-white z-10 fixed bg-ennustin-gray"
  />,
  <div
    className="min-h-screen pt-16 sm:ml-40 px-2 bg-white"
  >
    Ladataan...
  </div>,
]
`;
